vulyk/models/task_types.py

```diff
@@ -75,16 +76,20 @@ def import_tasks(self, tasks, batch):
 
         :raise: TaskImportError
         """
-        errors = (AttributeError, TypeError, ValidationError, OperationError)
+        errors = (AttributeError, TypeError, ValidationError, OperationError,
+                  AssertionError)
+        bulk = []
 
         try:
             for task in tasks:
-                self.task_model.objects.create(
+                assert isinstance(task, dict)
+
+                bulk.append(self.task_model(
                     id=sha1(json.dumps(task).encode('utf8')).hexdigest()[:20],
                     batch=batch,
                     task_type=self.type_name,
-                    task_data=task,
-                )
+                    task_data=task))
+            self.task_model.objects.insert(bulk, load_bulk=True)
```

Comment:
What will happen if such task already exists? Whole bulk import will die and transaction will be rolled back?